python/ray/gcs_utils.py

```diff
@@ -35,6 +36,7 @@
     "TablePubsub",
     "Task",
     "TaskTableData",
+    "RayResource",
```

Comment:
maybe rename `RayResource` to `ResourceTableData`?

Reply:
Done.